front/src/modules/trainschedule/components/Timetable/Timetable.tsx

```diff
@@ -104,6 +108,8 @@ const Timetable = ({
     });
   }, [currentDepartureDates]);
 
+  const VirtualizerRef = useRef<VirtualizerHandle>(null);
```

Comment:
Nit:

- I don't think we need this ref?
- This is a regular variable, so should start with a lowercase letter

Reply:
yes, not needed. I removed it.